gocd/pipelines/snuba-stable.yaml

```diff
@@ -60,6 +60,67 @@ pipelines:
+                                    --container-name="api" \
+                                    --container-name="consumer" \
+                                    --container-name="errors-consumer" \
+                                    --container-name="errors-replacer" \
+                                    --container-name="events-subscriptions-executor" \
+                                    --container-name="events-subscriptions-scheduler" \
+                                    --container-name="generic-metrics-counters-consumer" \
+                                    --container-name="generic-metrics-counters-subscriptions-executor" \
+                                    --container-name="generic-metrics-counters-subscriptions-scheduler" \
+                                    --container-name="generic-metrics-distributions-consumer" \
+                                    --container-name="generic-metrics-distributions-subscriptions-executor" \
+                                    --container-name="generic-metrics-distributions-subscriptions-scheduler" \
+                                    --container-name="generic-metrics-sets-consumer" \
+                                    --container-name="generic-metrics-sets-subscriptions-executor" \
+                                    --container-name="generic-metrics-sets-subscriptions-scheduler" \
+                                    --container-name="loadbalancer-outcomes-consumer" \
+                                    --container-name="metrics-consumer" \
+                                    --container-name="metrics-counters-subscriptions-scheduler" \
+                                    --container-name="metrics-sets-subscriptions-scheduler" \
+                                    --container-name="metrics-subscriptions-executor" \
+                                    --container-name="outcomes-billing-consumer" \
+                                    --container-name="outcomes-consumer" \
+                                    --container-name="querylog-consumer" \
+                                    --container-name="replacer" \
+                                    --container-name="snuba-admin" \
+                                    --container-name="transactions-consumer-new" \
+                                    --container-name="transactions-subscriptions-executor" \
+                                    --container-name="transactions-subscriptions-scheduler" \
+                                    && /devinfra/scripts/k8s/k8sdeploy.py \
+                                    --context="gke_${GCP_PROJECT}_${GKE_REGION}-${GKE_CLUSTER_ZONE}_${GKE_CLUSTER}" \
+                                    --label-selector="service=snuba,is_canary=true" \
+                                    --image="us.gcr.io/sentryio/snuba:${GO_REVISION_SNUBA_REPO}" \
+                                    --type="cronjob" \
```

Comment:
Do canaries apply to cron jobs? If so, how do we prevent duplicate jobs?

Reply:
Ah good catch, yeah I don't have any intention of adding canaries to cronjobs